src/index.ts

```diff
@@ -129,3 +129,50 @@ export * from './matrix/matrixPQN';
 export * from './matrix/matrixZRescale';
 
 export * from './utils/createSequentialArray';
+
+export type ArrayType = number[] | Float64Array | Float32Array | Uint16Array;
+export interface Zone {
+  fromIndex?: number;
+  toIndex?: number;
+  to: number;
+  nbPoints?: number;
+  from: number;
+}
+
+export interface Point {
+  x: number;
+  y: number;
+  index?: number;
+}
+export interface XYPoints {
+  x: ArrayType;
+  y: ArrayType;
+}
+
+export interface DataX {
+  x: ArrayType;
+  re: ArrayType;
+  im: ArrayType;
+}
+
+export interface Reim {
```

Comment:
DataReIm is define a little bit higher in the file and should be used instead of Reim. The definition of DataReIm should be corrected as well so that both are ArrayType

Reply:
Resolved